resources/views/components/usuario/form.blade.php

```diff
@@ -0,0 +1,157 @@
+<form action="{{ $action }}" method="post">
+
+  @csrf
+
+  @isset($usuario)
+    @method('PUT')
```

Comment:
trazer do controller o valor correto da diretiva, se atualizar "PUT", se criar forçar "POST"